public/app/plugins/panel/singlestat2/SingleStatPanel.tsx

```diff
@@ -0,0 +1,66 @@
+// Libraries
+import React, { PureComponent, CSSProperties } from 'react';
+
+// Types
+import { SingleStatOptions } from './types';
+
+import { processSingleStatPanelData, DisplayValue, PanelProps } from '@grafana/ui';
+import { config } from 'app/core/config';
+import { getDisplayProcessor } from '@grafana/ui';
+import { ProcessedValuesRepeater } from './ProcessedValuesRepeater';
+
+export const getSingleStatValues = (props: PanelProps<SingleStatOptions>): DisplayValue[] => {
```

Comment:
this should be SingleStatBaseOptions as this is used by gauge & bargauge panels and looks like SingleStatOptions is specific for the singlestat panel (confusing naming here but can't think of better named for SingleStatBaseOption and etc) so the moment we add a unique option for singlestat2 panel it will complain.

Reply:
yup.  done